### PR TITLE
created common next button and fixed the next button in app language selection

### PR DESCRIPTION
**Pull request by renukaj07**
…tion

### CODE REVIEW

File: src/components/common/NextButton.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#common-next-button{
+    justify-content: space-between;
+    padding-left: min(10px,2vw);
+    position: fixed;
```
Comment (bhanugummana) [inline src/components/common/NextButton.css:4]: Don't set position fixed here

File: src/components/common/NextButton.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ const NextButton: React.FC<{
+  onClicked:React.MouseEventHandler<HTMLButtonElement>,
+  disabled:boolean
+  
+}> = ({onClicked,disabled}) => {
+  return (
+    <button
+    id='common-next-button'
+    disabled={disabled}
+    color="light"
```
Comment (bhanugummana) [inline src/components/common/NextButton.tsx:16]: remove color here

File: src/pages/AppLangSelection.css [hunk 3]
```diff
@@ -32,17 +34,20 @@
 }
 
 .app-lang-next-button {
+    justify-content: space-between;
+    margin-top: -145px;
```
Comment (bhanugummana) [inline src/pages/AppLangSelection.css:38]: Don't use px

File: src/pages/AppLangSelection.css [hunk 4]
```diff
@@ -32,17 +34,20 @@
...
+    padding-left: min(0px,2vw);
```
Comment (bhanugummana) [inline src/pages/AppLangSelection.css:39]: here it will always take zero

**Review by bhanugummana — CHANGES_REQUESTED** [review]

File: src/pages/EditStudent.tsx [hunk 5]
```diff
@@ -156,13 +157,18 @@ const EditStudent = () => {
         msg={t("Please create your child profile").toString()}
       />
       <div className="content">
+
         {stage === STAGES.NAME && (
           <StudentNameBox
             studentName={studentName!}
             onValueChange={setStudentName}
             onEnterDown={isNextButtonEnabled() ? onNextButton : () => {}}
           />
```
Comment (bhanugummana) [inline src/pages/EditStudent.tsx:166]: Fix Two className with the same id(content)

**Review by bhanugummana — CHANGES_REQUESTED** [review]

File: src/pages/EditStudent.tsx (hunk 5 above)
Comment (renukaj07) [reply src/pages/EditStudent.tsx:166]: > Fix Two className with the same id(content)

<IonPage id={"Edit-student-page"}> 

this one right?

Comment (renukaj07) [reply src/pages/EditStudent.tsx:166]: > No, This one
> 
> ```
> <div className="content">
> ```
> 
> I can see it 2 times

done, now see

**Review by bhanugummana — APPROVED** [review]

**Pull request merged**